OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -1152,6 +1152,29 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 					}
 				}
 
+				// 'Selectable' boolean was removed from selectable trait.
+				if (engineVersion < 20150619)
+				{
+					if (depth == 1 && node.Value.Nodes.Exists(n => n.Key == "Selectable"))
+					{
+						var selectable = node.Value.Nodes.FirstOrDefault(n => n.Key == "Selectable");
+						if (node.Key == "Selectable" && selectable.Value.Value == "false")
+							node.Key = "SelectableRemoveMe";
+
+						// To cover rare cases where the boolean was 'true'
+						if (node.Key == "Selectable" && selectable.Value.Value == "true")
+							node.Value.Nodes.Remove(selectable);
+					}
+
+					if (depth == 0 && node.Value.Nodes.Exists(n => n.Key == "SelectableRemoveMe"))
+						node.Value.Nodes.RemoveAll(n => n.Key == "SelectableRemoveMe");
+						Console.WriteLine("The 'Selectable' boolean has been removed from the Selectable trait.");
```

Comment:
You forgot the braces here. Found out by accident when MonoDevelop corrected the indentions. Fixed it in https://github.com/OpenRA/OpenRA/pull/8412.